packages/catalog-process/package.json

```diff
@@ -29,6 +29,7 @@
     "@types/express": "4.17.21",
     "@types/node": "20.3.1",
     "@types/uuid": "9.0.8",
+    "date-fns": "3.3.1",
```

Comment:
Shouldn't be needed now that date utils are in `commons`

Reply:
Done with commit [c30b776](https://github.com/pagopa/interop-be-monorepo/pull/520/commits/c30b7765eaf90007889bb29aae469dbfcf4bdb85)